source/deployment-guide/server/air-gapped-deployment.rst

```diff
@@ -1,428 +1,99 @@
+Optional supporting services
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. **Create a kubernetes secret for registry authentication**:
-
-   .. code-block:: bash
-
-      kubectl create secret docker-registry regcred \
-        --docker-server=registry.example.com:5000 \
-        --docker-username=your_username \
-        --docker-password=your_password \
-        --docker-email=your_email@example.com
-
-2. **Reference the secret in pod specifications**:
-
-   .. code-block:: yaml
-
-      apiVersion: v1
-      kind: Pod
-      metadata:
-        name: mattermost-pod
-      spec:
-        containers:
-        - name: mattermost
-          image: registry.example.com:5000/mattermost/mattermost-enterprise-edition:latest
-        imagePullSecrets:
-        - name: regcred
-
-3. **For Helm deployments**, specify the registry in ``values.yaml``:
-
-   .. code-block:: yaml
-
-      image:
-        repository: registry.example.com:5000/mattermost/mattermost-enterprise-edition
-        tag: latest
-        pullPolicy: IfNotPresent
-      
-      imagePullSecrets:
-        - name: regcred
-
-Configure Docker to use private image registries
--------------------------------------------------
-
-Configure Docker on all hosts to trust and use your private registry.
-
-Docker daemon configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-1. **Add your registry to Docker's trusted registries**:
-
-   Edit or create ``/etc/docker/daemon.json``:
-
-   .. code-block:: json
-
-      {
-        "insecure-registries": ["registry.example.com:5000"]
-      }
-
-   For registries using self-signed certificates:
-
-   .. code-block:: bash
-
-      mkdir -p /etc/docker/certs.d/registry.example.com:5000
-      cp domain.crt /etc/docker/certs.d/registry.example.com:5000/ca.crt
-
-2. **Restart Docker daemon**:
-
-   .. code-block:: bash
-
-      systemctl restart docker
-
-3. **Test the configuration**:
-
-   .. code-block:: bash
-
-      docker pull registry.example.com:5000/mattermost/mattermost-enterprise-edition:latest
-
-Set up a private Debian package mirror
------------------------------------------
-
-A local Debian mirror allows you to maintain packages for system updates and dependencies. In this case, the mirror will be used to provide packages for Mattermost server to debian-based hosts.
-
-Ideally the air-gapped network already has a local mirror available. If not, you can set up a local mirror such as Aptly or debmirror.
-
-.. tab:: Use Aptly
-
-   1. **Install Aptly** (on an internet-connected machine):
-
-      .. code-block:: bash
-
-         apt-get update
-         apt-get install aptly gnupg
-
-   2. **Create GPG key for signing packages**:
-
-      .. code-block:: bash
-
-         gpg --gen-key
+Consider gathering additional resources if you plan to enable these optional components:
 
-   3. **Create a mirror configuration**:
+- :doc:`Mattermost Calls </administration-guide/configure/calls-deployment>`: For self-hosted audio and screensharing capabilities
```

Comment:
Maybe need a brief note "The Calls Offloader depends on specific Mattermost Docker image.  See <insert doc reference>".